### PR TITLE
Add has permission class decorator

### PR DESCRIPTION
**Pull request by Maillol**
Decorator that set permission for each `aiohttp.web.View` method.

``` python
@class_has_permission(prefix='booking')
class BookingView(web.View):
     def post(self):
         """
         Require `booking.create` permission
         """

     def get(self):
         """
         Require `booking.read` permission
         """

     def put(self):
         """
         Require `booking.update` permission
         """

     def patch(self):
         """
         Require `booking.update` permission
         """

     def delete(self):
         """
         Require `booking.delete` permission
         """
```
Note: This pull request is based on https://github.com/aio-libs/aiohttp-security/pull/148

### CODE REVIEW

File: aiohttp_security/api.py [hunk 1]
```diff
@@ -147,6 +156,41 @@ def wrapper(fn):
     return wrapper
 
 
+def class_has_permission(permission_prefix, context=None):
```
[the remark this answers is not in the document]
Comment (Maillol) [reply aiohttp_security/api.py]: Yes we can change the name, but if the `has_permission` decorator is deprecated (https://github.com/aio-libs/aiohttp-security/issues/169) I think that we cannot use this API to have set global permission.

[the remark this answers is not in the document]
Comment (kxepal) [reply aiohttp_security/api.py]: I was thought that not just `has_permission` was deprecated, but the decorator-way to check permissions is deprecated. Wouldn't this one face the same fate?

**Comment by asvetlov** [comment]
If we'll decide to deprecate decorators the PR should be revisited.
Let's postpone merging for now.

**Comment by asvetlov** [comment]
Thank you for PR.
Unfortunately, I've chosen another solution: no more decorators but `check_permission` explicit call.

**Pull request closed without merging**